Views/General/Curriculum/CourseCardView.swift

```diff
@@ -12,30 +12,49 @@ struct CourseCardView: View {
     @State var showPopUp = false
 
     var body: some View {
-        VStack {
-            Text(Course.startTimes[course.startTime - 1].clockTime)
-            Spacer()
-
-            Text(course.name)
-                .lineLimit(nil)
-            Text(course.classPositionString)
-            if course.startTime != course.endTime {
-                Divider()
-                Text(course.classIDString)
-                Text(course.classTeacherName)
-
+        VStack (spacing:3){
+            HStack {
```

Comment:
There's a modifier named `hStackLeading()` inside `SwiftUIExtension.swift` to simplify this if you want.